packages/react-devtools-extensions/webpack.backend.js

```diff
@@ -56,16 +57,32 @@ module.exports = {
     },
   },
   optimization: {
-    minimize: false,
+    minimize: !__DEV__,
+    minimizer: [
+      new TerserPlugin({
+        terserOptions: {
+          compress: {
+            unused: true,
+            dead_code: true,
+          },
+          mangle: {
+            keep_fnames: true,
+          },
+          format: {
+            comments: false,
+          },
+        },
+        extractComments: false,
+      }),
+    ],
   },
   plugins: [
     new Webpack.ProvidePlugin({
       process: 'process/browser',
     }),
     new Webpack.DefinePlugin({
-      __DEV__: true,
+      __DEV__,
       __PROFILE__: false,
-      __DEV____DEV__: true,
```

Comment:
Seems like a mistake. `react-devtools-extensions/webpack.config.js` doesn't define that var either.